src/diffusers/models/attention.py

```diff
@@ -206,6 +219,13 @@ def _set_attention_slice(self, slice_size):
         self.attn1._slice_size = slice_size
         self.attn2._slice_size = slice_size
 
+    def _set_use_memory_efficient_attention_xformers(self, use_memory_efficient_attention_xformers: bool):
+        if is_xformers_available():
+            self.attn1._use_memory_efficient_attention_xformers = use_memory_efficient_attention_xformers
+            self.attn2._use_memory_efficient_attention_xformers = use_memory_efficient_attention_xformers
```

Comment:
(nit) Could we maybe do some more checks here? E.g. what if `xformers` is available but the kernels are not compiled or available? What if Pytorch is < 1.12 - does `xformers` throw an error in this case?

Reply:
In my experience it does at runtime but this might a bit late indeed. Happy to update it

Reply:
I think the pytorch version check should probably done inside the function `is_xformers_available()` as it is a package dependency. What do you think ?

Reply:
> I think the pytorch version check should probably done inside the function is_xformers_available() as it is a package dependency. What do you think ?
Makes sense to me!